ngraph/frontend/onnx_import/src/op/reduce.cpp

```diff
@@ -66,19 +126,36 @@ namespace ngraph
 
                 template <typename OpType>
                 std::shared_ptr<ngraph::Node>
-                    make_ng_reduction_op(const Node& node, const Output<ngraph::Node>& ng_input)
+                    make_ng_reduction_op(const Node& node,
+                                         const Output<ngraph::Node>& ng_input,
+                                         bool axes_as_attr = true)
```

Comment:
I really dislike function signatures with bool flags. In code usage they later look like `do_something(true, false, false, true)`, which is very unreadable. In this case however, I'm not sure what to suggest as an alternative.

Reply:
I've considered breaking it down into two separate functions, but I don't know which is clearer.